changefeedtrigger/__init__.py

```diff
@@ -0,0 +1,94 @@
+import time
+import json
+from subprocess import Popen, PIPE
+from azure.storage.blob.changefeed import ChangeFeedClient
+from azure.storage.blob import BlobServiceClient
+import azure.functions as func
+from datetime import timedelta, datetime
+import logging
+from predicate import where
+
+
+def multidimensional_filtering(cf_client, b_filter, blob_client, cont_name, c_filter, e_filter):
```

Comment:
This function does a lot more than just multidim filtering. Also functions names should be verb-ish.

See more in-depth article about naming things https://dzone.com/articles/naming-conventions-from-uncle-bobs-clean-code-phil .

Reply:
I renamed the methods and split it up into three so that it makes more sense-- also gives users the option to use the program and not filter anything